docs/rfc/0001-cli.md

```diff
@@ -0,0 +1,318 @@
+# RMT CLI
+
+We would like to have a single entry-point script named `rmt` to provide CLI to RMT functions.
+
+```
+o rmt
```

Comment:
we now that it will clash on some systems.

we need a different name of the cli tool itself, I think.

`rmt-cli`  ?

Reply:
We would like it, doesn't mean that we can have it :-)
But yeah, it is likely that we'll need to consider a rename.